tests/agents/test_final_answer.py

```diff
@@ -19,22 +19,20 @@
 import numpy as np
 from PIL import Image
 
-from transformers import is_torch_available, load_tool
+from transformers import is_torch_available
 from transformers.agents.agent_types import AGENT_TYPE_MAPPING
+from transformers.agents.default_tools import FinalAnswerTool
 from transformers.testing_utils import get_tests_dir, require_torch
 
-from .test_tools_common import ToolTesterMixin
-
 
 if is_torch_available():
     import torch
 
 
-class FinalAnswerToolTester(unittest.TestCase, ToolTesterMixin):
+class FinalAnswerToolTester(unittest.TestCase):
```

Comment:
Expected to remove the tool tester mixin here?

Reply:
I had removed it because FinalAnswerTool does not need the input/output type testing that other tools do need, but indeed we can keep it (and it may be useful if some tests are added to `ToolTes) so I added it back!